website/blog/2024-09-30-detox-copilot-is-out.md

```diff
@@ -0,0 +1,219 @@
+---
+
+# Introducing Detox Copilot: Natural Language Testing for Detox
+
+We're excited to announce the release of **Detox Copilot**, a groundbreaking feature that brings natural language testing to Detox. With Detox Copilot, you can now write end-to-end tests using plain English commands, making test creation more intuitive and accessible than ever before.
+
+## Revolutionizing Testing with Natural Language
+
+Detox Copilot leverages advanced Large Language Models (LLMs) to interpret natural language instructions and translate them into Detox actions and assertions. This means you can describe your test scenarios in everyday language, aligning perfectly with **Behavior-Driven Development (BDD)** principles.
+
+### Why Natural Language Testing?
+
+- **Improved Collaboration**: Teams can collaborate more effectively, as tests are written in plain language understandable by developers, QA engineers, and non-technical stakeholders alike.
+- **Faster Test Creation**: Reduce the time spent writing and maintaining complex test scripts.
+- **Enhanced Test Coverage**: Encourage more comprehensive testing by lowering the barrier to writing tests.
+
+![Detox Copilot in action GIF](/img/blog/copilot-demo.gif)
+
+## Key Features of Detox Copilot
+
+### Write Tests in Plain English
+
+Detox Copilot allows you to write tests using natural language commands. Each step corresponds to a specific action or assertion within your app.
+
+**Example:**
+
+```javascript
+describe('User Registration Flow', () => {
+  it('should allow a user to register successfully', async () => {
+    await copilot.perform(
+      'Launch the app',
+      'Tap on the "Sign Up" button',
+      'Enter "Jane Doe" into the "Name" field',
+      'Enter "jane.doe@example.com" into the "Email" field',
+      'Enter "password123" into the "Password" field',
+      'Tap on the "Register" button',
+      'Verify that the "Welcome, Jane!" message is displayed'
+    );
+  });
+});
+```
+
+### Seamless Integration with Detox
+
+Detox Copilot is built into Detox and requires no additional installation. Simply initialize it in your test setup, and you're ready to start writing natural language tests.
+
+### LLM-Agnostic Design
+
+While Detox Copilot uses LLMs to interpret instructions, it is designed to be LLM-agnostic. This means you can connect it to your preferred language model service, offering flexibility and future-proofing your testing strategy.
+
+## Getting Started with Detox Copilot
+
+### Step 1: Initialize Detox Copilot
+
+Initialize Detox Copilot in your test setup by implementing a `PromptHandler` that connects to your LLM service. This abstraction allows Detox Copilot to communicate with any LLM you choose, promoting versatility across different testing frameworks.
+
+In our [Testing with Copilot guide](https://wix.github.io/Detox/docs/next/guide/testing-with-copilot), you can find implementation examples and more detailed descriptions for this part.
```

Comment:
`/next/...`?

Reply:
yeah, `next/` is the most up-to-date docs (on master). I'll try to use the latest published version instead.

Reply:
replaced with the current version docs